Configuration/TypoScript/Backend/cleanup.ts

```diff
@@ -0,0 +1,4 @@
+module.tx_ats.settings.cleanup {
+  deleteUnusedApplicationsAfter = 30 days
+  deleteInactiveUsersAfter = 2 years
```

Comment:
How about `unusedApplicationsLifetime` and `inactiveUsersLifetime`? BTW, wouldn't _unfinished_ be a more suitable term for the applications?

Reply:
You're right. Changed.